synapse/handlers/e2e_keys.py

```diff
@@ -358,7 +359,7 @@ def _exception_to_failure(e):
     # Note that some Exceptions (notably twisted's ResponseFailed etc) don't
     # give a string for e.message, which json then fails to serialize.
     return {
-        "status": 503, "message": str(e.message),
+        "status": 503, "message": str(e.args[0]),
```

Comment:
don't exceptions have a `message`?

Reply:
not on python 3, `.message` is removed

Reply:
Can we just do `str(e)` tbh, otherwise it feels a bit too awful